File: 프로그래머스/0/120868. 삼각형의 완성조건 （2）/삼각형의 완성조건 （2）.js
function solution(sides) {
    let answer = 0
    const max = Math.max(...sides)
    const min = Math.min(...sides)
    
    for (let i = max - min + 1; i <= max; i++) {
        answer++
    }
    
    for (let i = max + 1; i < max + min; i++) {
        answer++
    }
    return answer
}

File: 프로그래머스/0/120868. 삼각형의 완성조건 （2）/삼각형의 완성조건 （2）.py
def solution(sides):
    answer = 0
    max_side = max(sides)
    min_side = min(sides)
    for x in range(max_side - min_side + 1, max_side + 1):
        answer += 1
    for x in range(max_side + 1, max_side + min_side):
        answer += 1
    return answer